doc/index.md

```diff
@@ -173,32 +224,9 @@ print(bit.band(1, 3)) --> 1
 
 Deals with obtaining a connection to an HTTP server.
 
-
-### `negotiate(socket, options, timeout)` <!-- --> {#http.client.negotiate}
-
-  - `socket` is a cqueues socket object
-
-  - `options` is a table containing:
-
-	  - `tls` (boolean|userdata, optional): the `SSL_CTX*` to use, or a boolean to indicate the default TLS context.  
-		defaults to `true`.
-
-		  - `true` indicates to use the default TLS settings, see [*http.tls*](#http.tls) for information.
-		  - `false` means do not negotiate TLS
-
-	  - `version` (nil|1.0|1.1|2): HTTP version to use.
-		  - `nil`: attempts HTTP 2 and falls back to HTTP 1.1
-		  - `1.0`
-		  - `1.1`
-		  - `2`
-
-	  - `h2_settings` (table, optional): HTTP 2 settings to use.  
-		See [*http.h2_connection*](#http.h2_connection) for details
-
-
 ### `connect(options, timeout)` <!-- --> {#http.client.connect}
 
-Creates a new connection to an HTTP server.
+This function returns an open connection to an HTTP server. Once a connection has been opened, a stream can be created to start a request/response exchange. Please see [`h1_stream.new_stream`](h1_stream.new_stream) and [`h2_stream.new_stream`](h2_stream.new_stream) for more information about creating requests.
```

Comment:
It's not necessarily "open"; only after calling `:connect()` on the result is the connection truly "open".

Though, we haven't defined what "open" means...... I assumed it would be the underlying socket, but perhaps not?